### DISCUSSION TITLE
fedimint integration

### DISCUSSION THREAD

**Discussion by mikestaub** [GaloyMoney/galoy#2034] [Ideas]
Are there plans on working with the [fedimint](https://github.com/fedimint/fedimint) team?

These projects appear to have great synergy.

**Comment by nicolasburtey** [comment]
what type of integration would you like to see?

**Comment by mikestaub** [comment]
It's not clear to me yet, but it seems like fedimint's approach is currently the best way to reach mass adoption as most people will simply not feel comfortable with self-custody now if ever. I imagine a sort of "credit union in a box" service that could combine the privacy scalability and trust-model of fedimint, with the UX and tooling of Galoy.

**Comment by nicolasburtey** [comment]
For info, Galoy is already been used as a "credit union in a box" by many communities around the world. Probably the main difference is that Fedimint has an encrypted ledger, Galoy has a normal double entry accounting ledger. 

If one community eventually want to work with financial institution (to on-ramp/off-ramp from fiat to bitcoin) probably those institutions would be more comfortable working with an union bank which has a more traditional approach in term of users and ledger. On the other end, for community where privacy is of upmost importance (the first example that come in mind are activists), Fedimint is probably the right approach.

I'm not familiar with the details of the Fedimint stack, but I would assume the rest of the stack is probably very similar to ours (custody in multisig, payment over lightning)

**Reply by nicolasburtey** [reply]
@mikestaub I've converted this one to a discussion

**Reply by mikestaub** [reply]
I see. So the analogy is like using Whatsapp vs Signal. I am also concerned with the legality of using fedimint, especially considering what happened to the tornado cash developers.

**Reply by nicolasburtey** [reply]
I think a comparison between Slack and Signal would be more accurate.
